release/release.go

```diff
@@ -117,3 +124,10 @@ type GcpImage struct {
 	Family  string `json:"family,omitempty"`
 	Name    string `json:"name,omitempty"`
 }
+
+// IBMCloudObject represents an IBMCloud/PowerVS cloud object - which is a cos bucket image
+type IBMCloudObject struct {
```

Comment:
By the same argument as for `images` and `Images`, shouldn't we call this `IBMCloudImage`?